bookmarks/templates/settings/general.html

```diff
@@ -60,6 +60,15 @@ <h2>Profile</h2>
             Disabling this feature will hide all previously shared bookmarks from other users.
           </div>
         </div>
+        <div class="form-group">
```

Comment:
Let's move this up to maybe below "Bookmark date format" , so that bookmark display options are somewhat grouped together.

Reply:
Did changes as requested, doing other changes before pushing updates